docs: clarify debug environment variables (#9527)
ipfs/kubo: docs/environment-variables.md

## `IPFS_FUSE_DEBUG`

> Enables fuse debug logging. By default, fuse debug logging is disabled. Any value (even "") activates it.

Reviewer: ```suggestion
If SET, enables fuse debug logging.

Default: false
```
The `Default:` is a convention most fields share, let's keep it.

## `YAMUX_DEBUG`

> Enables debug logging for the yamux stream muxer. By default, yamux debug logging is disabled. Any value (even "") activates it.

Reviewer: ```suggestion
If SET, enables debug logging for the yamux stream muxer.

Default: false
```
